sdk_extensions/aws_v1_support/src/main/java/io/opentelemetry/sdk/extensions/trace/aws/resource/DockerHelper.java

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
This class is not used after the removal of `container.id`. I would remove it and reintroduce it when your suggested spec changes are accepted :)

Reply:
I agree. let's remove it if we're not using it.

Reply:
removed DockerHelper and unit test from PR